src/test/cypress/integration/exercises/programming/ProgrammingExerciseManagement.spec.ts

```diff
@@ -45,26 +45,27 @@ describe('Programming Exercise Management', () => {
         it('Deletes an existing programming exercise', function () {
             cy.login(admin, '/').wait(500);
             navigationBar.openCourseManagement();
-            courseManagementPage.openExercisesOfCourse(course.title, course.shortName);
-            cy.get('[deletequestion="artemisApp.programmingExercise.delete.question"]').click();
+            courseManagementPage.openExercisesOfCourse(course.shortName);
+            cy.get('#delete-exercise').click();
```

Comment:
Is this the same button for all exercise types? Otherwise it should be #delete-programming-exercise

Reply:
I only added it for the programming and text exercises as part of this PR because I had to change them. Generally I think it makes more sense to keep this id generic. I don't see what benefits a more specific id for different exercise types would provide